static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -184,11 +197,12 @@ require.def(
             */
             playFrom: function (seconds) {
                 this._postBufferingState = MediaPlayer.STATE.PLAYING;
-                var seekingTo = this._range ? this._getClampedTimeForPlayFrom(seconds) : seconds;
-
+                var seekingTo = this._range ? this._getClampedTimeForPlayFrom(seconds) : seconds;             
+                
                 switch (this.getState()) {
                     case MediaPlayer.STATE.BUFFERING:
-                        this._deferSeekingTo = seekingTo;
+//                        this._deferSeekingTo = seekingTo;
```

Comment:
Change was necessary because SEF event handling is different compared to Device API (one event with params registered on plugin vs. seperate events). As a result direct assigning this._deferSeekingTo on BUFFERING state resulted in CATAL randomly staying in BUFFERING after seeking with no possibility to either change state or update currentTime until next seeking was perfomred by the user.